src/org/labkey/test/tests/SampleTypeRenameTest.java

```diff
@@ -0,0 +1,267 @@
+package org.labkey.test.tests;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.remoteapi.CommandException;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.categories.Daily;
+import org.labkey.test.components.ChartTypeDialog;
+import org.labkey.test.components.CustomizeView;
+import org.labkey.test.components.html.BootstrapMenu;
+import org.labkey.test.pages.experiment.UpdateSampleTypePage;
+import org.labkey.test.params.FieldDefinition;
+import org.labkey.test.params.experiment.SampleTypeDefinition;
+import org.labkey.test.util.DataRegionTable;
+import org.labkey.test.util.PortalHelper;
+import org.labkey.test.util.SampleTypeHelper;
+import org.labkey.test.util.TestDataGenerator;
+import org.labkey.test.util.exp.SampleTypeAPIHelper;
+import org.openqa.selenium.WebElement;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * <p>
+ *     Address <a href="https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=45861">Issue 45861: When sample types are renamed, the custom views associated with the original name are lost</a>.
+ * </p>
+ * <p>
+ *     A more general sample type rename test can be found in {@link SampleTypeTest#testSampleTypeNames}.
+ * </p>
+ */
+@Category({Daily.class})
+public class SampleTypeRenameTest extends BaseWebDriverTest
+{
+    private static final String PROJECT_NAME = "SampleType_Rename_Test";
+    private static final String SAMPLE_TYPE_NAME_INITIAL = "Sample Type To Be Renamed";
+    private static final String SAMPLE_TYPE_NAME_UPDATED = "A New Name For This Sample Type";
+    private static final String FIELD_STR = "Str";
+    private static final String FIELD_INT = "Int";
+    private static final Locator editButton = Locator.lkButton("Edit Type");
+
+    @Override
+    public List<String> getAssociatedModules()
+    {
+        return Arrays.asList("experiment");
+    }
+
+    @Override
+    protected String getProjectName()
+    {
+        return PROJECT_NAME;
+    }
+
+    @BeforeClass
+    public static void setupProject() throws IOException, CommandException
+    {
+        SampleTypeRenameTest init = (SampleTypeRenameTest) getCurrentTest();
+        init.doSetup();
+    }
+
+    private void doSetup() throws IOException, CommandException
+    {
+        PortalHelper portalHelper = new PortalHelper(this);
+        _containerHelper.createProject(PROJECT_NAME, null);
+        portalHelper.enterAdminMode();
```

Comment:
You don't need to explicitly goto Page Admin mode and exit, addWebPart does it

Reply:
Thanks for the pointer.
Updated code to:
`new PortalHelper(this).addWebPart("Sample Types");`